<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotationTuner</class>
 <widget class="QWidget" name="RotationTuner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="AngleTuner" name="tuner_x" native="true"/>
   </item>
   <item>
    <widget class="AngleTuner" name="tuner_y" native="true"/>
   </item>
   <item>
    <widget class="AngleTuner" name="tuner_z" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AngleTuner</class>
   <extends>QWidget</extends>
   <header>angletuner.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
